trs_filer/version.py

```diff
@@ -0,0 +1,3 @@
+"""Single source of truth for package version."""
+
+__version__ = '0.1.0'
```

Comment:
I think you can bump that version, don't you think? :)

Reply:
done!